Remotion/Web/Development.WebTesting/HostingStrategies/DockerHosting/IDockerClient.cs

```diff
@@ -53,11 +55,13 @@ public interface IDockerClient
     string Run (
         [NotNull] IDictionary<int, int> ports,
         [NotNull] IDictionary<string, string> mounts,
+        IDictionary<string, string> environmentVariables,
         [NotNull] string imageName,
         [CanBeNull] string? isolationMode,
         [CanBeNull] string? hostname,
         bool remove,
         [CanBeNull] string? entryPoint,
+        string? workingDirectory,
         [CanBeNull] string? args);
```

Comment:
please add the jetbrains annotations for consistency.

Reply:
Why do we even use these? The types already give all the information anyways

Reply:
This goes back to when ReSharper wasn't considering the nullable reference type attributes in .net code. We can eventually remove them, but I am lobbying for keeping things consistent. starting to mix in the same signature isn't helpful so it's easier to apply them consistently when extending existing code and doing a general clean-up later. We also need to then verify that nullability works correctly for ReSharper even when the feature is disabled in the c# compiler. And yes, projects keep NRT disabled due to exessive nullability inconsistencies while the ReSharper attributes are still helpful. And since the web test library is intended to help test automation engineers, we try to be as graceful as possible.